MAUI/Scheduler/reminders.md

```diff
@@ -0,0 +1,423 @@
+            {
+                new SchedulerReminder {TimeBeforeStart = new TimeSpan (0,4,0)},
+            }
+
+        };
+        Events.Add(normalAppointment);
+
+        // All Day Appointment
+        SchedulerAppointment allDayAppointment = new SchedulerAppointment()
+        {
+            StartTime = DateTime.Now.AddDays(1).AddMinutes(1),
+            EndTime = DateTime.Now.AddDays(1).AddHours(1),
+            Subject = "All Day Appointment",
+            Background = Brush.SkyBlue,
+            IsAllDay = true,
+            Reminders = new ObservableCollection<SchedulerReminder>
+            {
+                new SchedulerReminder {TimeBeforeStart = new TimeSpan (0,0,50)},
+            }
+
+        };
+        Events.Add(allDayAppointment);
+
+        // Recurrence Appointment
+        SchedulerAppointment recurrenceAppointment = new SchedulerAppointment()
+        {
+            Id = 1,
+            StartTime = DateTime.Now.AddDays(2).AddMinutes(1),
+            EndTime = DateTime.Now.AddDays(2).AddHours(1),
+            Subject = "Recurrence Appointment",
+            Background = Brush.LightGray,
+            RecurrenceRule = "FREQ=DAILY;COUNT=3",
+            Reminders = new ObservableCollection<SchedulerReminder>
+            {
+                new SchedulerReminder {TimeBeforeStart = new TimeSpan (0,0,40)},
+            }
+
+        };
+        Events.Add(recurrenceAppointment);
+ }
+{% endhighlight %}
+{% endtabs %}
+
+## Creating business object for reminders
+
+Create a business object class `Event` with mandatory fields `From,` `To,` and `EventName.` and also business object class `Reminder` with mandatory fields `TimeBeforeStart` and `IsDismissed`.
+
+{% tabs %}
+{% highlight c# tabtitle="Event.cs" %}
+/// <summary>
+/// Represents custom data properties.
+/// </summary>
+ public class Event
+    {
+        public DateTime From { get; set; }
+        public DateTime To { get; set; }
+        public bool IsAllDay { get; set; }
+        public string EventName { get; set; }
+        public string Notes { get; set; }
+        public string StartTimeZone { get; set; }
+        public string EndTimeZone { get; set; }
+        public Brush Color { get; set; }
+        public object RecurrenceId { get; set; }
+        public object Id { get; set; }
+        public string RecurrenceRule { get; set; }
+        public ObservableCollection<DateTime> RecurrenceExceptions { get; set; }
+        public ObservableCollection<Reminder> Reminders { get; set; }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+The [ReminderMapping] provides the mapping information about the [SchedulerReminder] properties to the [DataItem] object. `ReminderMapping` has the following properties.
+
+* [TimeBeforeStart]: Maps the property name of a custom class, which is equivalent for the [SchedulerReminder.TimeBeforeStart.]
+
+* [IsDismissed]: Maps the property name of a custom class, which is equivalent for the [SchedulerReminder.IsDismissed.]
```

Comment:
* [IsDismissed]: Maps the property name of a custom class, which is equivalent to the [SchedulerReminder.IsDismissed].